executor/update.go

```diff
@@ -163,6 +165,10 @@ func (e *UpdateExec) handleErr(colName model.CIStr, rowIdx int, err error) error
 func (e *UpdateExec) composeNewRow(rowIdx int, oldRow []types.Datum) ([]types.Datum, error) {
 	newRowData := types.CopyRow(oldRow)
 	for _, assign := range e.OrderedList {
+		colRange, isReplenish := e.replenishTbl[assign.Col.DBName.O+"-"+assign.Col.TblName.O]
```

Comment:
Is it an optimization? 
Can we remove this?

Reply:
mum..no..it's a bugfix not optimize, remove this will cause https://github.com/pingcap/tidb/pull/7177/files#diff-6d9a596d0d155b4342a598ba359efc8bR2985 failure